hathor/manager.py

```diff
@@ -155,10 +161,12 @@ def __init__(self, reactor: IReactorCore, peer_id: Optional[PeerId] = None, netw
         self.peer_discoveries: List[PeerDiscovery] = []
 
         self.ssl = ssl
-        self.server_factory = HathorServerFactory(self.network, self.my_peer, node=self, use_ssl=ssl)
-        self.client_factory = HathorClientFactory(self.network, self.my_peer, node=self, use_ssl=ssl)
+        self.server_factory = HathorServerFactory(self.network, self.my_peer, node=self, use_ssl=ssl,
+                                                  enable_sync_v1=enable_sync_v1, enable_sync_v2=enable_sync_v2)
+        self.client_factory = HathorClientFactory(self.network, self.my_peer, node=self, use_ssl=ssl,
+                                                  enable_sync_v1=enable_sync_v1, enable_sync_v2=enable_sync_v2)
         self.connections = ConnectionsManager(self.reactor, self.my_peer, self.server_factory, self.client_factory,
-                                              self.pubsub, self, ssl, rng=self.rng)
+                                              self.pubsub, self, ssl, whitelist_only=False, rng=self.rng)
```

Comment:
Do we need the whitelist_only parameter since it is not being used?

Reply:
This is being used on `PeerIdState._should_block_peer`, basically so that sync-v2 does not use a "whitelist-only" mode by default (which we can enable for some nodes) and still use whitelist-only on sync-v1.

If we migrate to netfilter this might not be needed, but currently (on this PR) it is in use.

Reply:
I got it. I agree that we should keep it until we migrate to netfilter later.